commandeer.nim

```diff
@@ -284,6 +284,26 @@ template subcommand*(identifier: untyped, subcommandName: string,
     statements
     currentMapping = subCommands[""]
 
+template subcommand*(identifier: untyped, subcommandNames: openArray[string],
+                    statements: untyped): untyped =
+  var identifier: bool = false
+  var namesOfSubcommands = @subcommandNames
+  var aSubcommandName = namesOfSubcommands.pop()
+  subCommands[aSubcommandName] = CommandLineMapping(
+    assigners: newSeq[Assigner](),
+    shortOptions: newTable[string, assignmentProc](),
+    longOptions: newTable[string, assignmentProc](),
+    activate: proc() =
+    identifier = true
+  )
+  currentMapping = subCommands[aSubcommandName]
+  statements
+  currentMapping = subCommands[""]
+  if len(namesOfSubcommands) > 0:
+    for subcommandName in namesOfSubcommands:
+      subCommands[subcommandName] = subCommands[aSubcommandName]
+
+
```

Comment:
Let's add a test to check your feature! You can just follow the example of the other ones.